### ISSUE TITLE
Dose the driver support linux4.6 kernel?

### ISSUE THREAD

**Issue by wwfdzh2015** [lwfinger/rtl8723au_bt#70]
hello
I want to driver the rtl8723bu BT part on linux4.6 kernel,but i can't find correctly the BT driver of this model correctly.Does it can support for it?What the branche?
Thank for your help!

**Comment by mochnaufals** [comment]
I also wonder if this driver support linux 5.4 kernel. Have you got any ideas or workarounds?

**Comment by lwfinger** [comment]
Yes, it should; however, in 5.4, the in-kernel drivers btusb and rtlusb should handle the device.
